platform/darwin/src/MGLLocationManager.m

```diff
@@ -38,7 +38,15 @@ - (CLLocationAccuracy)desiredAccuracy {
 }
 
 - (CLAuthorizationStatus)authorizationStatus {
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 140000
+    if (@available(iOS 14.0, *)) {
+        return self.locationManager.authorizationStatus;
+    } else {
+        return kCLAuthorizationStatusNotDetermined;
+    }
+#else
     return [CLLocationManager authorizationStatus];
+#endif
 }
 
 - (void)setActivityType:(CLActivityType)activityType {
```

Comment:
I know you didn't change this, but the function says set activity type, but you are just returning an existing activity type, that doesn't make sense

Reply:
The default implementation of `MGLLocationManager` is a wrapper of `CLLocationManager.